.github/workflows/website_tests.yml

```diff
@@ -0,0 +1,86 @@
+---
+name: Website Tests
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
This repo use gh_pages as the main branch, so I think that should be changed here too

Reply:
thanks a lot, I totally missed it

Reply:
for pull requests that should work though. maybe because it's marked as draft?